src/runtime/StaticStrings.h

```diff
@@ -206,6 +209,7 @@ namespace Escargot {
     F(exp)                        \
     F(extends)                    \
     F(expm1)                      \
+    F(false_)                     \
```

Comment:
We already define an atomic string of `false` as `stringFalse` in StaticStrings.